duckduckgo/iOS
# Update app to support xliff translations

Reviewer: Chris Brind
Asana: https://app.asana.com/0/361428290920652/391964084463161

**Description**:
Update the app to support exporting strings to an xliff file to allow us to translate the app.

Strings now live in their individual components as sharing them causes xliff duplicates. This is probably a better implementation anyway.

The convenience NSLocalizedString method that allowed us to reduce boilerplate comments and reuse the `key` for the `comment` was removed as the xliff needs the NSLocalizedString call to be at the point of String creation
```
func forKey(_ key: String) -> String {
   return NSLocalizedString(key, comment: key)
}
```

**Steps to test this PR**:
1. Open XCode
2. Select Editor, export for Localization
3. Ensure that this works smoothly and  produces an xliff file

###### Reviewer Checklist:

###### PR DRI Checklist:
